sqlconnect/internal/databricks/db.go

```diff
@@ -102,29 +96,17 @@ func NewDB(configJson json.RawMessage) (*DB, error) {
 					return fmt.Sprintf("SHOW TABLES IN `%[1]s` LIKE '%[2]s'", schema, base.EscapeSqlString(table))
 				}
 				cmds.ListColumns = func(catalog, schema, table base.UnquotedIdentifier) (string, string, string) {
-					if catalog == "" || !informationSchema {
+					if catalog == "" {
```

Comment:
`catalog` is being passed by caller, and caller would get this info from how customer has set config for profiles/retl source or warehouse destination? If yes, should we check once configs for these?

Reason being old logic used to depend on value of catalog as well as additional check of  informationschema. Majorly trying to be cautious a bit if indeed RudderStack UI enforces user to configure catalog info correctly.